src/test/java/collab/ModifyTest.java

```diff
@@ -27,60 +27,50 @@ public void modifyTest_returnEmployee() {
     @Test
     public void modifyTest_Name() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // name 변경 Test
       empDB.modifyItemByCondition("employeeNum", "17112609", "name", "TEST TEST");
 
-      assertTrue(empDB.searchItems("name","FB NTAWR").size()==0);
-      assertTrue(empDB.searchItems("name","TEST TEST").size()==1);
+      assertEquals(empDB.searchItems("name","FB NTAWR").size(),0);
+      assertTrue(empDB.searchItems("name","TEST TEST").get(0).getEmployeeNumber() == "17112609");
     }
     
     @Test
     public void modifyTest_Cl() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // cl 변경 Test
-      assertThrows(RuntimeException.class, () -> {
-        empDB.modifyItemByCondition("employeeNum", "17112609", "cl", "CL5");
-
-        assertTrue(empDB.searchItems("cl","CL4").size()==0);
-        assertTrue(empDB.searchItems("cl","CL5").size()==1);
-      });
+        empDB.modifyItemByCondition("employeeNum", "17112609", "cl", "CL1");
 
+        assertEquals(empDB.searchItems("cl","CL4").size(),0);
+        assertTrue(empDB.searchItems("cl","CL1").get(0).getEmployeeNumber() == "17112609");
+      
     }
     
     @Test
     public void modifyTest_PhoneNum() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // phoneNum 변경 Test
       empDB.modifyItemByCondition("employeeNum", "17112609", "phoneNum", "010-0000-0000");
-            
-      assertTrue(empDB.searchItems("phoneNum","010-5645-6122").size()==0);
-      assertTrue(empDB.searchItems("phoneNum","010-0000-0000").size()==1);
+
+      assertEquals(empDB.searchItems("phoneNum","010-5645-6122").size(),0);
+      assertTrue(empDB.searchItems("phoneNum","010-0000-0000").get(0).getEmployeeNumber() == "17112609");
     }
     
     @Test
     public void modifyTest_Birthday() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // birthday 변경 Test
       empDB.modifyItemByCondition("employeeNum", "17112609", "birthday", "19001101");
-            
-      assertTrue(empDB.searchItems("birthday","19860903").size()==0);
-      assertTrue(empDB.searchItems("birthday","19001101").size()==1);
+
+      assertEquals(empDB.searchItems("birthday","19860903").size(),0);
+      assertTrue(empDB.searchItems("birthday","19001101").get(0).getEmployeeNumber() == "17112609");
     }    
     
     @Test
     public void modifyTest_certi() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // certi 변경 Test
       empDB.modifyItemByCondition("employeeNum", "17112609", "certi", "EX");
       
-      assertTrue(empDB.searchItems("certi","PRO").size()==0);
-      assertTrue(empDB.searchItems("certi","EX").size()==1);
+      assertEquals(empDB.searchItems("certi","PRO").size(),0);
```

Comment:
assertEquals(기대값, 실제값)으로 쓰도록 정의되어 있네요.
실패했을 때 결과 표시도 Expected, Actual로 나오는 것으로 보아 이 형식에 맞추는 것이 좋을 것 같습니다.

```suggestion
      assertEquals(0, empDB.searchItems("certi","PRO").size());
```